### PR TITLE
[MI-127] OAuth Tokens

### PR DESCRIPTION
**Pull request by joseconsador**
Add OAuth Tokens resource and test

/cc @miogalang @jwswj @mmolina @atroche

### References
 - Jira link: https://zendesk.atlassian.net/browse/MI-127

### Risks
 - Low. We might not be able to access the OAuth Tokens endpoints using the client

### CODE REVIEW

File: src/Zendesk/API/Resources/OauthTokens.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Zendesk\API\Resources;
+
+use Zendesk\API\Traits\Resource\Delete;
+use Zendesk\API\Traits\Resource\Find;
+use Zendesk\API\Traits\Resource\FindAll;
+
+class OauthTokens extends ResourceAbstract
```
Comment (miogalang) [inline src/Zendesk/API/Resources/OauthTokens.php]: OAuthTokens

Comment (jwswj) [inline src/Zendesk/API/Resources/OauthTokens.php]: Agree

**Comment by jwswj** [comment]
:+1:

**Comment by miogalang** [comment]
:+1:

**Pull request merged**